.github/actions/setup-local-volume-provisioner/manifests/local-csi-driver/10_csidriver.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: storage.k8s.io/v1
```

Comment:
Do you think these manifests could be moved to replace `examples/common/provisioner` instead? We could use them in eks/gke examples with some tweaks.

Reply:
current examples are still using `examples/common/provisioner`. I wouldn't mind moving these there when existing examples are converted to the dynamic provisioner. But as a part of separate PR.